pkg/util/namespace.go

```diff
@@ -40,9 +45,18 @@ func CreateNamespace(client kubeclient.Interface, namespaceObj *corev1.Namespace
 
 // DeleteNamespace just try to delete the namespace.
 func DeleteNamespace(client kubeclient.Interface, namespace string) error {
```

Comment:
This is the common function that we can't couple with the logic of checking annotations.